src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
It's better to list out all the classes you import specifically. You can change the IntelliJ settings to avoid replacing all the classes with a wildcard (*).